scipy/linalg/__init__.py

```diff
@@ -121,8 +121,9 @@
    solve_sylvester - Solve the Sylvester matrix equation
    solve_continuous_are - Solve the continuous-time algebraic Riccati equation
    solve_discrete_are - Solve the discrete-time algebraic Riccati equation
+   solve_continuous_lyapunov - Solve the continous-time Lyapunov equation
    solve_discrete_lyapunov - Solve the discrete-time Lyapunov equation
-   solve_lyapunov - Solve the (continous-time) Lyapunov equation
```

Comment:
You can skip an entry by adding it to `REFGUIDE_AUTOSUMMARY_SKIPLIST` here:

https://github.com/scipy/scipy/blob/master/tools/refguide_check.py#L117